### PR TITLE
(GH-510) Fix for Choco new silentargs can't pass in args in the param…

### LINKED ISSUE #510: Choco new silentargs can't pass in args in the param=value format
Some applications like Operations Manager use a similar format to choco new to do pass in their arguments, e.g. USE_SETTINGS_FROM_AD={0|1} . (https://technet.microsoft.com/en-us/library/hh230736.aspx)

Creating a new package from template with silentarg="/qn /norestart USE_SETTINGS_FROM_AD=0" will create an emtpy string in the chocolateyInstall.ps1 file. I've tried quoting it several different ways (double quotes, single quotes, double quotes around single quotes, etc).
```powershell
silentArgs    = ""
```

This is using Chocolatey v0.9.10-beta1

### PR DESCRIPTION
**Pull request by sahebjade**
…=value format

Closes #510

### CODE REVIEW

**Comment by ferventcoder** [comment]
@sahebjade Have you had the opportunity to review our CONTRIBUTING document? We would ask that you read over and sign a CLA so that we can take contributions - https://github.com/chocolatey/choco/blob/master/CONTRIBUTING.md

Apologies, usually the CLA bot check shows up on PRs, but it appears it did not for this PR.

**Comment by ferventcoder** [comment]
Now it looks like it may be incorrectly flagging as signed. Looks like we may need to find another CLA host

File: src/chocolatey/infrastructure.app/commands/ChocolateyNewCommand.cs [hunk 1]
```diff
@@ -79,7 +79,7 @@ public void handle_additional_argument_parsing(IList<string> unparsedArguments,
 
             foreach (var unparsedArgument in unparsedArguments.or_empty_list_if_null())
             {
-                var property = unparsedArgument.Split(new[] {'='}, StringSplitOptions.RemoveEmptyEntries);
+                var property = unparsedArgument.Split(new[] { '=' }, 2, StringSplitOptions.RemoveEmptyEntries);
```
Comment (ferventcoder) [inline src/chocolatey/infrastructure.app/commands/ChocolateyNewCommand.cs]: Super simple. Love it.

**Comment by ferventcoder** [comment]
Followup @sahebjade, it appears the CLA has not yet been signed. I mentioned this above in https://github.com/chocolatey/choco/pull/513#issuecomment-166727946. The status is showing incorrectly here as I resent those through and it didn't pick up the original submitter of this request. If you can review and sign, we can move forward with this fix.

**Comment by sahebjade** [comment]
Sorry I was away from work. Now I have signed the CLA. Please can you move it further. Thanks.

**Pull request closed without merging**